scripts/extract.py

```diff
@@ -68,6 +78,10 @@ def extract_multi_process(self):
                 file=sys.stdout):
             self.faces.num_faces_detected += 1
             self.faces.faces_detected[os.path.basename(filename)] = faces
+            frame_no += 1
+            if frame_no == self.save_interval:
```

Comment:
As per earlier comment, self.save_interval will not necessarily exist, so reference it directly from arguments and make sure it is a passed argument: `if hasattr(self.args, "save_interval") and self.args.save_interval == frame_no`